.github/workflows/cut-nightly.yml

```diff
@@ -35,3 +35,10 @@ jobs:
         run: node --unhandled-rejections=strict build-system/release-workflows/cut-nightly.js
         env:
           GITHUB_TOKEN: ${{ secrets.AMPPROJECTBOT }}
+
+  create-issue-on-error:
+    needs: cut-nightly
+    if: failure()
+    uses: ampproject/amphtml/.github/workflows/create-issue-on-error.yml@main
```

Comment:
Does this exist?

Reply:
Aahahhahahaaaahahaha `git commit --all` doesn't add new filesssssssssss
(fixed)